### PR TITLE
August HB

### PR DESCRIPTION
**Pull request by elleobrien**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by elleobrien** [comment]
Some notes:

- I didn't include @shcheklein's dev.to posts, because I wasn't sure where to put them. They might be nice to add.
- I didn't include the mlops coffee podcast because I feel like I already have enough talks/videos here, but if there's a place it makes sense it could be good to add

File: content/blog/2020-08-10/august-20-dvc-heartbeat.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+href="https://medium.com/@karthik.vaithyanathan/using-continuous-machine-learning-to-run-your-ml-pipeline-eeeeacad69a3"
+title="Using Continuous Machine Learning to Run Your ML Pipeline"
+description="Vaithy Narayanan"
+link="medium.com"
+image="/uploads/images/2020-08-10/vaithy.jpg"/>
+
+- [Ryan Gross](https://www.linkedin.com/in/ryan-w-gross/), a VP at Pariveda
+  Solutions, blogged about the future of data governance and the lessons from
+  DevOps that might save the day. Honestly, you should probably start reading
+  for this cover image alone. ![](/uploads/images/2020-08-10/dataops.png)
```
Comment (shcheklein) [inline content/blog/2020-08-10/august-20-dvc-heartbeat.md:185]: separate line? 

also, please optimize the image if possible

File: content/blog/2020-08-10/august-20-dvc-heartbeat.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+Welcome to our August roundup of cool news, new releases, and recommended
+reading in the MLOps world!
+
+## News
+
+### CML release
+
+At the beginning of July, we went live with a new project: Continuous Machine
+Learning, or CML for short. If you hadven't heard, CML is an open-source toolkit
```
Comment (shcheklein) [inline content/blog/2020-08-10/august-20-dvc-heartbeat.md:31]: link to CML - site and/or github would be helpful

File: content/blog/2020-08-10/august-20-dvc-heartbeat.md (hunk 1 above)
Comment (elleobrien) [reply content/blog/2020-08-10/august-20-dvc-heartbeat.md]: shrunk it in revisions w/tinypng

File: content/blog/2020-08-10/august-20-dvc-heartbeat.md (hunk 2 above)
Comment (elleobrien) [reply content/blog/2020-08-10/august-20-dvc-heartbeat.md]: added

**Comment by elleobrien** [comment]
@shcheklein revisions have been pushed!

**Pull request merged**